packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -59,6 +67,11 @@ export function specTestIterator(configDirpath: string, testRunners: Record<stri
       }
 
       for (const testHandler of readdirSyncSpec(testRunnerDirpath)) {
+        if (
+          ["full_withdrawals", "partial_withdrawals", "bls_to_execution_change", "withdrawals"].includes(testHandler)
```

Comment:
Since this lists are growing large can you define constants at the top with the exclusion list or tests, forks, etc?